src/main/pages/setup-docker/docker-single-user.adoc

```diff
@@ -287,7 +298,7 @@ docker run -p 8080:8080 \
 
 *Ports*
```

Comment:
* -> section heading
* -> action title

Reply:
Changed to "Running Che on ports"

Reply:
That doesn't make sense. How about "Configuring port mapping for Che"? And "for Che" is not really needed. Up to you.